### PR TITLE
catch up to U gate introduction / Ux gate removal

### PR DESCRIPTION
**Pull request by ssmi1975**
I have tried updating the tutorial based on the comment in the slack. If this direction is fine, I will update the other pages.

--

Uゲート追加とのことで勉強しながら 005 を更新してみましたが、こういうのでよかったでしょうか？
よければ他の部分も更新お手伝いします。

### CODE REVIEW

**Comment by ssmi1975** [comment]
updated the tutorial 123 too.
I have omitted the `gamma` parameter of U-gates because it doesn't seem relevant. But it would be great if you could check it too.

File: tutorial/123_pea2_en.ipynb [hunk 1]
```diff
@@ -115,13 +115,13 @@
     "\n",
     "In this case, diagonal elements of $D$ are eigenvalues of $\\hat H$ and each column of $P$ are eigenvalues of $\\hat H$. Now, we're considering a 2x2 Hermitian with eigenvalues summing up to zero, so, elements of $D$ are $\\pm E$ when $\\hat H$ has an eigenvalue $E$.\n",
     "\n",
-    "Thus, we generate random real number $E$ and make $D$, then, generate random three real numbers $\\theta, \\phi, \\lambda$ and make $P$ using U3 gate, i.e. $P = \\mathrm{U3}(\\theta, \\phi, \\lambda)$.\n",
+    "Thus, we generate random real number $E$ and make $D$, then, generate random three real numbers $\\theta, \\phi, \\lambda$ and make $P$ using U gate, i.e. $P = \\mathrm{U}(\\theta, \\phi, \\lambda)$. Here, we ignore the parameter $\\gamma$ of U gate because it is irrelevant to eigenvalues.\n",
```
Comment (ssmi1975) [inline tutorial/123_pea2_en.ipynb:118]: is the last sentence OK?

**Review by ryuNagai — APPROVED** [review]
Thank you for commit! I think the changes are all OK.
I approve merging.

File: tutorial/123_pea2_en.ipynb (hunk 1 above)
Comment (ryuNagai) [reply tutorial/123_pea2_en.ipynb:118]: I think OK.
As a note, in the use of CU gate, the parameter gamma changes relative phase and could be unignorable.
In this tutorial, there are no problem.

**Pull request merged**